### ISSUE TITLE
[BUG] keywords not highlighted

### ISSUE THREAD

**Issue by noeleont** [hedyorg/hedy#4126]
**Describe the bug**
Keywords aren't always highlighted in quizzes. 

**Paste the Hedy code & level**

For example https://hedycode.com/hedy/3#quiz first question, only `print` is highlighted.

**Add a screenshot (optional)**

<img width="1440" alt="Screenshot 2023-03-14 at 15 51 41" src="https://user-images.githubusercontent.com/2394624/225039749-c021cd1f-b96a-4c8d-a3d6-ba129ff494ba.png">

**What machine and browser you were using (optional)**
If the issue concerns things in the website, let us know:
- What computer you are using (Windows, Mac, Linux?)
Mac
- What browser you were using (Chrome, Edge, Safari)
Safari, also tested in Chrome

**Comment by Felienne** [comment]
Thanks for noticing @noeleont!!

This is a content issue, we can fix it by replacing ``` ```` with `` in the quiz yamls. Three backticks highlight programs but not keywords.

If we change
```
            -   option: |
                    ```
                    {ask}
                    ```
```

into 
```
            -   option: |
                    `{ask}
```

It will work              

<img width="1127" alt="image" src="https://user-images.githubusercontent.com/1003685/225325664-0e0b7d1d-70ce-44ea-b301-79b5ea32e171.png">

**Comment by noeleont** [comment]
Ok I see, but why does it work with \`print\` then? I just looked up the translation and there it's 
\```
print
\```
as well

**Comment by Felienne** [comment]
If you want to, you can totally help fix this. If not we will pick it up!

**Comment by noeleont** [comment]
So for I only provided translations which in this case probably won't scale. I guess one has to 

- go through every entry in content/quizzes/<LANG>.yaml 
    - check if `option` contains code -> NO leave it
        - YES -> check if code is correct -> YES leave it
                - NO -> save it as \`code\` instead of \```code\```

can you point me in the right direction for checking if the code is correct based on the level? Maybe I can take a look but can't guarantee it.

**Comment by Felienne** [comment]
Yes sadly it will be a bit of manual checking :( 

But the steps are a bit different:

* go through every entry in content/quizzes/.yaml
    * check if `option` contains code 
        * NO -> leave it
        * YES -> check if code is only one keyword
            * NO -> Leave it (for a full cone snippet ``` works)
            * YES -> save it as `code` instead of ```code```

**Comment by noeleont** [comment]
Damn that's rough, can't we use hedy itself to do this checking? I mean if we enter the keyword only on the website we get an error message what code gets executed there?

**Comment by Felienne** [comment]
> Damn that's rough, can't we use hedy itself to do this checking? I mean if we enter the keyword only on the website we get an error message what code gets executed there?

Yeah, true, such is the sad reality of having a lot of content in a lot of languages. But also it does not have to be perfect everywhere. Fixing this question only already is a step towards better quality!
